README.md

```diff
@@ -2,33 +2,55 @@
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/7243b915eb214102be03f7c7b4dd2681)](https://app.codacy.com/gh/physarumAdv/Physarum_webGL?utm_source=github.com&utm_medium=referral&utm_content=physarumAdv/Physarum_webGL&utm_campaign=Badge_Grade_Settings)
 
-Visualization written on JS library THREE.js of a clever mould Physarum Polycephalum
-
-You can view this project [here](http://pavtiger.silaeder.ru/physarum)
-
-## Physarum Polycephalum 
-In nature is a mould that minimizes energy on
-growth process and food transfer. This way it builds the shortes and most optimal ways between points
-
-![Screenshot](readme_images/physarum.gif)
-
-## Cube
-Now growth is visualized at a single cube but mesh can be imported as a .gltf 3D format
-![Screenshot](readme_images/cube.jpg)
-
-## Growth
-![Screenshot](readme_images/mold.jpg)
-
-## Change color of the mold
-Using a slider at the top-right side of the screen you can change the color of the mold to anything you want while simulation is running
-![Screenshot](readme_images/color.jpg)
+This is a visualizer for Physarum Polycephalum's growth written in JavaScript using THREE.js and Node.js libraries.
+We recommend using [this](https://github.com/physarumAdv/minds_crawl) simulator. <br/>
+You can also visit this project on a website: [pavTiger.silaeder.ru](http://pavtiger.silaeder.ru)
+
+## Visualizer
+There are two visualizer modes:
+* **Offline**: Show a precalculated growth simulation
+
+* **Online**: If a simulator is working then it will show live growth. <br/>
+     ![](readme_images/interface_small.png)
```

Comment:
Not a readme issue, just asking about the feature described here (you or I can start a new issue if you think it is worthy). Is it possible for any regular user to switch any server between these modes? If yes, should it be so?

Reply:
I didn't really understand you. What do you mean when you say "to switch a server between these nodes"?

Reply:
There is a selector in controls which allows me to switch between online and offline modes. So, is any user able to switch the whole server between these modes using controls?